composer.json

```diff
@@ -12,10 +12,11 @@
   "minimum-stability": "stable",
   "require": {
     "php": ">=7.0",
+    "ext-dom": "*",
     "aws/aws-sdk-php": "^3.0",
     "roave/security-advisories": "dev-master",
     "simplesamlphp/composer-module-installer": "^1.1.5",
-    "simplesamlphp/simplesamlphp": "~1.17.7 || ~1.18.5",
+    "simplesamlphp/simplesamlphp": "~1.17.7 || ~1.18.6",
```

Comment:
Why the bump from 1.18.5 to 1.18.6? Was there anything about this change that would prevent 1.18.5 from working?

Reply:
I had made the 1.18.5 change directly on develop so I could get ssp-base to pull in 1.18.5. Then a day or two later 1.18.6 was released to address a security concern. To answer your question, probably not, but I wanted to be sure I was getting the latest.